lms/djangoapps/instructor/views/api.py

```diff
@@ -2629,8 +2636,20 @@ def spoc_gradebook(request, course_id):
         for student in enrolled_students
     ]
```

Comment:
@pdpinch django pagination api divides result into chunks that  it call pages, 
My aim was to divide view into pages. So that user dont need to scroll a huge list.
I have a question what is maximum number of students normally a edX course have? 
If you guys think student list will be huge then may be i need to change my logic here.

Edx code example of pagination https://github.com/edx/edx-platform/blob/062e979e0ed14161d0f882091a06e032ca76bb3e/openedx/core/lib/api/paginators.py#L10

https://docs.djangoproject.com/en/1.8/topics/pagination/